yt/docker/ya-build/query-tracker/package.json

```diff
@@ -0,0 +1,84 @@
+{
+    "meta": {
+        "name": "query-tracker",
+        "maintainer": "YT team",
+        "description": "Query Tracker image built with yt client libraries from source files",
+        "version": "local-{revision}",
+    },
+    "params": {
+        "format": "docker",
+        "docker_target": "query-tracker",
+        "docker_build_arg": {
+            "PYTHON_BUILD_BASE": "build-ytsaurus-python-binaries",
+        },
+    },
+    "include": [
+      "yt/docker/ya-build/base/package.json",
```

Comment:
I remember we discussed that this ends up breaking the param definitions above. Idk what you figured out here, but anything changed in how `ya package` has to be called (for example, if one is always required to specify docker_target now), please add it to the corresponding READMEs. 

Also, let's create issues in yatool, if something is broken.

Reply:
https://github.com/yandex/yatool/issues/18